### DISCUSSION TITLE
展开动态内容，会导致动态专栏预览过长时去掉全文字样

### DISCUSSION THREAD

**Discussion by u2shana** [the1812/Bilibili-Evolved#4475] [功能建议 / Ideas]
![image](https://github.com/the1812/Bilibili-Evolved/assets/14146754/73a8ba25-88f7-4c17-a3b9-98349b2685b3)
上图是正常情况下的专栏动态预览
下图是开启展开动态内容模块的预览
![image](https://github.com/the1812/Bilibili-Evolved/assets/14146754/c5ce5aa7-8c49-471e-a4a2-c6d28c971e01)

可以看到开启后去掉了全文字样，加上动态专栏和普通动态样式几乎一样，这样很容易以为结束了而不点开错过里面的详细专栏内容

**Comment by the1812** [comment]
不是几乎一样, 是完全一样, 一样到我不知道该怎么区分这是不是专栏了...

如果用 FeedsCardsManager 后续打的类型标记 `data-type` 区分, 对专栏取消强制展开后也没有按钮, 因为一开始用 CSS 强制展开全文后会让 b 站认为不需要添加全文按钮.

先试试用那个粗体标题做判断吧, 不确定靠不靠谱.
